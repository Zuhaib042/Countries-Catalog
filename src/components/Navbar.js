import React from 'react';
import { FaMicrophone } from 'react-icons/fa';
import { BsFillGearFill } from 'react-icons/bs';
// import {FaChevronLeft} from 'react-icons/fa';
// import { BsArrowRightCircle } from 'react-icons/bs';
const Navbar = () => {
  return (
    <div className="bg-light text-white py-4 flex items-center justify-between">
      <p className="text-2xl ml-2 md:ml-8">2022</p>
      <p className="text-white font-bold text-2xl">Countries Catalogue</p>
      <div className="flex text-lg gap-2 mr-2 md:mr-8">
        <FaMicrophone />
        <BsFillGearFill />
      </div>
    </div>
  );
};

export default Navbar;
