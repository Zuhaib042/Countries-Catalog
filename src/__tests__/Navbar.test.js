import { render, screen } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Navbar test', () => {
  it('should render header correctly', () => {
    render(
      <Router>
        <Navbar />
      </Router>,
    );
    const countriesTag = screen.getByText(/countries/i);
    expect(countriesTag).toMatchSnapshot();
  });
});
