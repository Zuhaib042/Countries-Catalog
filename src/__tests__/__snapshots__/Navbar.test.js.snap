// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar test should render header correctly 1`] = `
<p
  class="text-white font-bold text-2xl"
>
  Countries Catalogue
</p>
`;
